src/fitnesse/resources/javascript/fitnesse.js

```diff
@@ -115,6 +115,37 @@ $(document)
         }
     });
 
+// When checking purgeGlobal then change the href elements to send true
+$(document)
+    .on('change', '.testHistory #purgeGlobal', function() {
+        const purgeGlobal = "&purgeGlobal=true";
+        let elems = $("a[href^='?responder=purgeHistory']");
```

Comment:
`elem` is not changed anywhere, is it?

```suggestion
        const elems = $("a[href^='?responder=purgeHistory']");
```